django-stubs/apps/registry.pyi

```diff
@@ -24,7 +24,7 @@ class Apps:
     def get_app_config(self, app_label: str) -> AppConfig: ...
     # it's not possible to support it in plugin properly now
     def get_models(self, include_auto_created: bool = ..., include_swapped: bool = ...) -> list[type[Model]]: ...
-    def get_model(self, app_label: str, model_name: str | None = ..., require_ready: bool = ...) -> type[Any]: ...
+    def get_model(self, app_label: str, model_name: str | None = ..., require_ready: bool = ...) -> type[Model]: ...
```

Comment:
IIRC it was a deliberate choice to use `Any` instead of `Model` here.

Does this cause issues with pyright?

Reply:
I thought that it caused some issues but cannot replicate now 🤷 
Let me add a comment and undo
@intgr , what was the reason for using `Any` there?

Reply:
I think the reasoning is here: https://github.com/typeddjango/django-stubs/pull/814#issuecomment-1301957498